pkg/workload/workload.go

```diff
@@ -40,6 +40,10 @@ var (
 	admissionManagedConditions = []string{kueue.WorkloadQuotaReserved, kueue.WorkloadEvicted, kueue.WorkloadAdmitted}
 )
 
+const (
+	QueuingPolicyIsNever = "Never"
```

Comment:
This constant should be defined in apis/kueue/v1beta1/workload_types.go instead

Reply:
also consider a new type for it, see example in https://github.com/kubernetes/api/blob/master/batch/v1/types.go#L115-L138